docs/background/kubernetes/gardener/garden-linux.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Gardener in Cleura Cloud runs its worker and control plane nodes on Garden Linux.
+---
+# Garden Linux
+
+In a {{k8s_management_service}} cluster, worker nodes run [Garden Linux](https://gardenlinux.io), which is a [Linux distribution](https://en.wikipedia.org/wiki/Linux_distribution) based on [Debian GNU/Linux](https://debian.org/).
+
+Garden Linux provides an automated, auditable build pipeline.
+Its snapshot [releases](https://github.com/gardenlinux/gardenlinux/releases) are automatically generated from GitHub Actions workflows.
```

Comment:
It's probably just me, but I have to wonder: Why are we providing these pieces of info regarding automated, auditable build pipelines, and snapshots generated from GitHub Actions workflows?

Reply:
I am not saying otherwise. All I'm saying is that this piece of info came in somewhat unexpectedly. At least this is how I felt, as a reader.

Reply:
Fair. Then it's probably unneeded. I've dropped that paragraph.